src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -30,7 +30,7 @@ azure-mgmt-cognitiveservices==6.2.0
 azure-mgmt-compute==13.1.0
 azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==1.5.0
-azure-mgmt-containerregistry==3.0.0rc14
+azure-mgmt-containerregistry==3.0.0rc15
```

Comment:
pls make sure you run live test to regress all ACR tests.

Reply:
When will azure-mgmt-containerregistry 3.0.0 be released officially? We recently got some requirements from Homebrew to only use stable versions of the dependencies.

Reply:
@fengzhou-msft,  azure-mgmt-containerregistry is a multi-api package including both ga and prewiew api versions. We can release a new non preview/rc package in the near future and submit a new PR. Does this sound reasonable?
Meanwhile, could you please share the context for why Homebrew would only allow stable versions?

Reply:
HI @yugangw-msft pls see the conversation https://github.com/Homebrew/homebrew-core/pull/61520#discussion_r492907434. We do let them know rc sdk version might contain both stable and preview API, and CLI has preview commands by https://github.com/Homebrew/homebrew-core/pull/61520#issuecomment-698424333. while from our end, we also want to let people aware to try to use stable version as possible.